src/logic/proposals/encoding/encoding.test.ts

```diff
@@ -67,7 +79,33 @@ describe('encoding incorrect params', () => {
     expect(goRes.error).toEqual(new EncodedEvmScriptError('parameters', 'Make sure parameters is a valid JSON array'));
   });
 
-  it('empty parameters', async () => {
+  it('wrong number of parameters', async () => {
```

Comment:
I think these tests would read a bit better if they were named along the lines of:
```ts
it('returns an error when the number of parameters do not match', ...
it('returns an error for zero length parameters and a non-empty target signature', ...
...
```

Reply:
It reads better when you open this in IDE since there is more context provided by the wrapping `describe`. I've tried to improve the readability using `it` and `test` so that it reads better. But, personally I am not too keen on this.